### DISCUSSION TITLE
Github Actions Failed because of billing

### DISCUSSION THREAD

**Discussion by xianruizhong** [community/community#34404] [Actions]
Hi,

I used to have a github action in a branch and it worked, but today when I merged it to main, it showed "The job was not started because recent account payments have failed or your spending limit needs to be increased. Please check the 'Billing & plans' section in your settings."

I checked my billing, nothing has exceeded the limit. Can someone help me with this? Thanks!

**Comment by antonblr** [comment]
I don't think anyone can help here without looking into your account/billing. I'd suggest checking the docs first, to see if there is anything obvious:

* [Billing for GitHub Actions](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions#included-storage-and-minutes)
* [View your Actions usage](https://docs.github.com/en/billing/managing-billing-for-github-actions/viewing-your-github-actions-usage)
* [Spending limits for Actions](https://docs.github.com/en/billing/managing-billing-for-github-actions/managing-your-spending-limit-for-github-actions)

Otherwise contact the support - https://support.github.com/contact/account
